bot.py

```diff
@@ -295,6 +295,16 @@ def cmd_uptime(match: Match[str]) -> Response:
     return UptimeResponse()
 
 
+@handle_message('!pep')
+def cmd_pep(match: Match[str]) -> Response:
+    _, _, msg = match.groups()
+    _, _, rest = msg.partition(' ')
+    return MessageResponse(
+        match,
+        f'https://www.python.org/dev/peps/pep-{rest.zfill(4)}/',
```

Comment:
this can (unfortunately) expose the token currently, if someone ran:

```
!pep {config}
```

it would show my stream token D:

there's an `_esc(...)` function above which is helpful for sanitizing user input

Reply:
Alright, lemme try that quick

Reply:
`Undefined variable: '_esc'`

Reply:
Python error is `NameError: name '_esc' is not defined`